feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/BookmarkIcon.kt

```diff
@@ -25,27 +25,26 @@ import io.github.droidkaigi.confsched2023.feature.sessions.R.raw
 @Composable
 fun BookmarkIcon(
     contentDescription: String,
-    onBookmarkClickStatus: Boolean?,
+    onBookmarkClickStatus: Boolean,
     onReachAnimationEnd: () -> Unit,
     modifier: Modifier = Modifier,
 ) {
     val lottieComposition by rememberLottieComposition(RawRes(raw.add_to_bookmark_lottie))
-    val animationRange = 0.1f..1f
     var isPlaying by remember { mutableStateOf(false) }
 
-    val progress by animateLottieCompositionAsState(
+    val state = animateLottieCompositionAsState(
         composition = lottieComposition,
         isPlaying = isPlaying,
         restartOnPlay = true,
     )
 
     LaunchedEffect(onBookmarkClickStatus) {
-        if (onBookmarkClickStatus != null) {
+        if (onBookmarkClickStatus) {
```

Comment:
update `isPlaying` to true only when `onBookmarkClickStatus` is `true`.
this is for not launcing animation when `onBookmarkClickState` is `false`.